qutebrowser/keyinput/keyutils.py

```diff
@@ -274,8 +277,6 @@ def _parse_special_key(keystr):
         ('command', 'meta'),
         ('cmd', 'meta'),
         ('mod1', 'alt'),
-        ('less', '<'),
```

Comment:
We shouldn't break backwards compatibility, many people are already binding with `<alt+greater>` (and I personally think it's less confusing than `<alt+>>`, even if we can use that syntax.

Reply:
I've reverted the commit in 120aa82fe2a133d6df3ed06b76dfa3342377ec73 :smile: